frontstage/views/session.py

```diff
@@ -0,0 +1,14 @@
+from datetime import datetime, timezone
+
+from flask import Blueprint, jsonify, request
+
+from frontstage.common.authorisation import jwt_authorization
+
+session_bp = Blueprint("session", __name__)
+
+
+@session_bp.route("/expires-at", methods=["GET"])
+@jwt_authorization(request, refresh_session=False)
+def session_expires_at(session):
+    expires_at = session.get_expires_in()
```

Comment:
This is could do with fixing at some stage, expires_in is not right, however I can't just change this due to the affect it would have on live jwt session on a deploy to PROD